config/pipeline.yaml

```diff
@@ -57,6 +57,18 @@ _anchors:
       compiler: gcc-10
       defconfig: x86_64_defconfig
 
+  kbuild-gcc-10-armel-android-mainline: &kbuild-gcc-10-armel-android-mainline-job
+    <<: *kbuild-gcc-10-armel
+    rules:
+      tree:
+      - 'android'
+
+  kbuild-gcc-10-armel-android-mainline-imx: &kbuild-gcc-10-armel-android-mainline-imx-job
+    <<: *kbuild-gcc-10-armel-android-mainline-job
+    params:
+      <<: *kbuild-gcc-10-armel-params
+      defconfig: imx_v6_v7_defconfig
+
```

Comment:
Please move those to the `jobs` section (`_anchors` is only used for, well, anchors, and not parsed as actual pipeline items), right below `kbuild-gcc-10-armel` would be good.

Reply:
done, please check